qiskit_experiments/library/calibration/fine_amplitude.py

```diff
@@ -32,74 +32,76 @@
-    repetitions, e.g. [1, 2, 3, 5, 7, ...] to ensure that the ideal points are on the equator
-    of the Bloch sphere. Note the presence of two repetitions which allows us to prepare the
-    excited state. Therefore, add_xp_circuit = True is not needed in this case.
-
-    **Summary of experiment options**
-
-    * repetitions: A list with the number of times the gate of interest will be repeated.
-    * schedule: The schedule of the gate that will be repeated.
-    * add_sx: A boolean which if set to True will add a square-root of X before the repetitions
-      of the gate of interest. Set this to True if you are calibrating gates with an ideal
-      rotation angle per gate of :math:`\pm\pi`. The default value is False.
-    * add_xp_circuit: A boolean which if set to True will cause the experiment to run an
-      additional circuit with an X gate and a measurement. This prepares the excited state
-      and is typically crucial to get the correct sign for the magnitude of the error in
-      the rotation angle. The default value is True.
-    * sx_schedule: Allows users to set a schedule for the square-root of X gate.
-
-    **Summary of analysis options**
-
-    * angle_per_gate: The ideal angle per repeated gate. The user must set this options.
-    * phase_offset: A phase offset for the analysis. This phase offset will be :math:`\pi/2`
-      if the square-root of X gate is added before the repeated gates. This is decided for
-      the user in :meth:`set_schedule` depending on whether the sx gate is included in the
-      experiment.
-
-    Users can call :meth:`set_schedule` to conveniently set the schedule and the corresponding
-    experiment and analysis options.
+    # section: overview
+
+        The :class:`FineAmplitude` calibration experiment repeats N times a gate with a pulse
+        to amplify the under-/over-rotations in the gate to determine the optimal amplitude.
+        The circuits that are run have a custom gate with the pulse schedule attached to it
+        through the calibrations. The circuits are therefore of the form:
+
+        .. parsed-literal::
+
+                       ┌─────┐       ┌─────┐ ░ ┌─┐
+                  q_0: ┤ Cal ├─ ... ─┤ Cal ├─░─┤M├
+                       └─────┘       └─────┘ ░ └╥┘
+            measure: 1/════════ ... ════════════╩═
+                                                0
+
+        Here, Cal is the name of the gate which will be taken from the name of the schedule.
+        The user can optionally add a square-root of X pulse before the Cal gates are repeated.
+        This square-root of X pulse allows the analysis to differentiate between over rotations
+        and under rotations in the case of pi-pulses. Importantly, the resulting data is analyzed
+        by a fit to a cosine function in which we try and determine the over/under rotation given
+        an intended rotation angle per gate which must also be specified by the user. The steps

```

Comment:
Can you use `# section: example` for this block? It would be great if you could add another example for set schedule in there.

Reply:
This should cover it: 1e9e478